<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>535</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lCircleX1">
         <property name="text">
          <string>x1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tCircleX1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="lCircleY1">
         <property name="text">
          <string>y1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tCircleY1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="lCircleR">
         <property name="text">
          <string>r   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tCircleR"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>471</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lRecX1">
         <property name="text">
          <string>x1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecX1"/>
       </item>
       <item>
        <widget class="QLabel" name="lRecY1">
         <property name="text">
          <string>y1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecY1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lRecX2">
         <property name="text">
          <string>x2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecX2"/>
       </item>
       <item>
        <widget class="QLabel" name="lRecY2">
         <property name="text">
          <string>y2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecY2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lRecX3">
         <property name="text">
          <string>x3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecX3"/>
       </item>
       <item>
        <widget class="QLabel" name="lRecY3">
         <property name="text">
          <string>y3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecY3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lRecX4">
         <property name="text">
          <string>x4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecX4"/>
       </item>
       <item>
        <widget class="QLabel" name="lRecY4">
         <property name="text">
          <string>y4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tRecY4"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_53">
       <item>
        <widget class="QLabel" name="lTriangleX1_14">
         <property name="text">
          <string>x1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleX1_14"/>
       </item>
       <item>
        <widget class="QLabel" name="lTriangleY1_14">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>y1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleY1_14"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_54">
       <item>
        <widget class="QLabel" name="lTriangleX2_14">
         <property name="text">
          <string>x2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleX2_14"/>
       </item>
       <item>
        <widget class="QLabel" name="lTriangleY2_14">
         <property name="text">
          <string>y2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleY2_14"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_55">
       <item>
        <widget class="QLabel" name="lTriangleX3_14">
         <property name="text">
          <string>x3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleX3_14"/>
       </item>
       <item>
        <widget class="QLabel" name="lTriangleY3_14">
         <property name="text">
          <string>y3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tTriangleY3_14"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>511</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Rysuj">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rysuj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>375</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usun</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>360</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolizja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1031</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>90</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>22</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>widget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>widget_2</zorder>
   <zorder>Rysuj</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_2</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>comboBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
